spring-cloud-dataflow-module-deployer-spi/src/main/java/org/springframework/cloud/dataflow/module/deployer/ModuleDeployer.java

```diff
@@ -69,7 +69,7 @@
 	 *
 	 * @param id id for the module this status is for
 	 *
-	 * @return module deployment status
+	 * @return module deployment status or {@literal null} if this deployer knows nothing about the given deployment
```

Comment:
Is this accurate? I think we're supposed to return `unknown` for unknown deployments.